### PR TITLE
Ignore contentless lesson directories

### PR DESCRIPTION
**Pull request by encukou**
Fixes: https://github.com/pyvec/naucse_render/issues/15

### CODE REVIEW

**Comment by encukou** [comment]
Sorry for the mess.
I skipped OS X tests because can't figure out how to handle Brew's switch to Python 3. Will track that here: https://github.com/pyvec/naucse_render/issues/17

File: test_naucse_render/fixtures/test_content/lessons/testcases/ignored_lesson/README [hunk 1]
```diff
@@ -0,0 +1,2 @@
+This directory doesnot have info.yml, and is therefore not included the list
```
Comment (frenzymadness) [inline test_naucse_render/fixtures/test_content/lessons/testcases/ignored_lesson/README]:
```suggestion
This directory does not have info.yml and is therefore not included the list
```

**Comment by encukou** [comment]
> Do not forget to change the version to 1.3 also in setup.py.

I'm planning to release with https://github.com/pyvec/naucse_render/pull/12. Or do you want to start using this now?

**Comment by frenzymadness** [comment]
> I'm planning to release with #12. Or do you want to start using this now?

Not right now. We still have more than a week before we start deploying the PyData courses.

**Comment by encukou** [comment]
Merged with the wrong summary :(
But it's in.

**Pull request merged**